app/test/interactors/game-room.js

```diff
@@ -8,7 +8,7 @@ import interactor, {
 
 import AppInteractor from './app';
 
-@interactor class ToastInteractor {
+@interactor export class ToastInteractor {
```

Comment:
Let's just `export { ToastInteractor }` at the bottom next to the default export. That will also avoid having to adjust eslint.